neon_diana_utils/helm_charts/neon/neon-iris-websat/values.yaml

```diff
@@ -0,0 +1,27 @@
+serviceName: neon-core-iris-websat
+subdomain: iris
```

Comment:
I'm not sure this is referenced anywhere looking at the `neon-iris` chart for comparison